### PR TITLE
Refactor using class

### PR DESCRIPTION
**Pull request by Addisu87**
- Refactoring the application of awesome books using class.

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -6,74 +6,86 @@ const addBtn = document.getElementById('add-btn');
...
+  static remove() {
+    booksArr = booksArr.filter((book) => +book.id !== +this.parentElement.id);
+    this.parentElement.remove();
 
-  localStorage.setItem('Books', JSON.stringify(booksArr));
-}
+    booksArr.forEach((book, i) => {
+      book.id = i;
+      bookList.children[i].id = i;
+    });
```
Comment (ahmetbozaci) [inline assets/js/index.js:29]: - [ ] Books array should inside the class. Please change it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=You%20can%20use%20objects%20and%20arrays%2C%20but%20do%20it%20inside%20classes.)

File: assets/js/index.js [hunk 2]
```diff
@@ -6,74 +6,86 @@ const addBtn = document.getElementById('add-btn');
...
   if (titleBookInput.value !== '' && authorBookInput.value !== '') {
-    AddBook();
-    clearField();
+    const newBook = new Book(
+      booksArr.length,
+      titleBookInput.value,
+      authorBookInput.value,
+    );
+    newBook.Add();
+    Book.clearField();
```
Comment (ahmetbozaci) [inline assets/js/index.js:87]: - [ ] I can't add a book. Please fix the error.

image

**Review by ahmetbozaci — COMMENTED** [review]
# STATUS: Changes Required  ♻️


 @Addisu87 @Jord4N-AC  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi Team; @Addisu87  and @Jord4N-AC , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**